release/api/v1alpha1/artifacts.go

```diff
@@ -1,54 +1,66 @@
 package v1alpha1
 
-func (vb *VersionsBundle) Manifests() map[string][]Manifest {
-	return map[string][]Manifest{
+func (vb *VersionsBundle) Manifests() map[string][]*Manifest {
+	return map[string][]*Manifest{
 		"cluster-api-provider-aws": {
-			vb.Aws.Components,
-			vb.Aws.ClusterTemplate,
-			vb.Aws.Metadata,
+			&vb.Aws.Components,
+			&vb.Aws.ClusterTemplate,
+			&vb.Aws.Metadata,
 		},
 		"core-cluster-api": {
-			vb.ClusterAPI.Components,
-			vb.ClusterAPI.Metadata,
+			&vb.ClusterAPI.Components,
+			&vb.ClusterAPI.Metadata,
 		},
 		"capi-kubeadm-bootstrap": {
-			vb.Bootstrap.Components,
-			vb.Bootstrap.Metadata,
+			&vb.Bootstrap.Components,
+			&vb.Bootstrap.Metadata,
 		},
 		"capi-kubeadm-control-plane": {
-			vb.ControlPlane.Components,
-			vb.ControlPlane.Metadata,
+			&vb.ControlPlane.Components,
+			&vb.ControlPlane.Metadata,
+		},
+		"cert-manager": {
+			&vb.CertManager.Manifest,
 		},
 		"cluster-api-provider-docker": {
-			vb.Docker.Components,
-			vb.Docker.ClusterTemplate,
-			vb.Docker.Metadata,
+			&vb.Docker.Components,
+			&vb.Docker.ClusterTemplate,
+			&vb.Docker.Metadata,
 		},
 		"cluster-api-provider-vsphere": {
-			vb.VSphere.Components,
-			vb.VSphere.ClusterTemplate,
-			vb.VSphere.Metadata,
+			&vb.VSphere.Components,
+			&vb.VSphere.ClusterTemplate,
+			&vb.VSphere.Metadata,
 		},
 		"cluster-api-provider-cloudstack": {
-			vb.CloudStack.Components,
-			vb.CloudStack.Metadata,
+			&vb.CloudStack.Components,
+			&vb.CloudStack.Metadata,
+		},
+		"cluster-api-provider-tinkerbell": {
+			&vb.Tinkerbell.Components,
+			&vb.Tinkerbell.ClusterTemplate,
+			&vb.Tinkerbell.Metadata,
 		},
 		"cilium": {
-			vb.Cilium.Manifest,
+			&vb.Cilium.Manifest,
 		},
 		"kindnetd": {
-			vb.Kindnetd.Manifest,
+			&vb.Kindnetd.Manifest,
 		},
 		"eks-anywhere-cluster-controller": {
-			vb.Eksa.Components,
+			&vb.Eksa.Components,
 		},
 		"etcdadm-bootstrap-provider": {
-			vb.ExternalEtcdBootstrap.Components,
-			vb.ExternalEtcdBootstrap.Metadata,
+			&vb.ExternalEtcdBootstrap.Components,
+			&vb.ExternalEtcdBootstrap.Metadata,
 		},
 		"etcdadm-controller": {
-			vb.ExternalEtcdController.Components,
-			vb.ExternalEtcdController.Metadata,
+			&vb.ExternalEtcdController.Components,
+			&vb.ExternalEtcdController.Metadata,
+		},
+		"eks-distro": {
+			&Manifest{URI: vb.EksD.Components},
+			&Manifest{URI: vb.EksD.EksDReleaseUrl},
```

Comment:
This will work for downloads, however, I don't think the values in the manifest will be updated, so ekds will still point to the original urls.
Or am I missing something?

Reply:
thanks, good catch, updated it